### PR TITLE
[Fusion vNext] Add more tests and fix minor issues

### PR DESCRIPTION
**Pull request by tobias-tengler**

### CODE REVIEW

File: src/HotChocolate/Fusion-vnext/test/Fusion.Execution.Tests/FusionTestBase.cs [hunk 1]
```diff
@@ -14,14 +14,21 @@ protected static CompositeSchema CreateCompositeSchema()
...
-
         var rewriter = new InlineFragmentOperationRewriter(compositeSchema);
-        var rewritten = rewriter.RewriteDocument(doc, null);
+        var rewritten = rewriter.RewriteDocument(request, null);
 
         var planner = new OperationPlanner(compositeSchema);
         return planner.CreatePlan(rewritten, null);
     }
+
+    protected static async Task MatchSnapshotAsync(DocumentNode request,RootPlanNode plan)
```
Comment (michaelstaib) [inline src/HotChocolate/Fusion-vnext/test/Fusion.Execution.Tests/FusionTestBase.cs:26]: should we add the original request document to the RootPlanNode and then integrate it into the yaml?

Comment (michaelstaib) [reply src/HotChocolate/Fusion-vnext/test/Fusion.Execution.Tests/FusionTestBase.cs:26]: This would simplify the snapshot

Comment (tobias-tengler) [reply src/HotChocolate/Fusion-vnext/test/Fusion.Execution.Tests/FusionTestBase.cs:26]: I think having the rewritten / normalized document produced by the inline fragment rewriter would make more sense, but yes 👍

Comment (michaelstaib) [reply src/HotChocolate/Fusion-vnext/test/Fusion.Execution.Tests/FusionTestBase.cs:26]: OK ... I will do a quick PR

**Pull request merged**